tooling/Common.props

```diff
@@ -0,0 +1,35 @@
+<Project>
+  <!-- Default configuration and platform when not present -->
+  <PropertyGroup>
+    <Configuration Condition="'$(Configuration)' == ''">Debug</Configuration>
+    <Platform Condition="'$(Platform)' == ''">AnyCPU</Platform>
+  </PropertyGroup>
+
+  <!-- Build to central artifacts folder instead of scattering bin and obj throughout the solution tree -->
+  <PropertyGroup>
+    <UseArtifactsOutput>true</UseArtifactsOutput>
+    <ArtifactsPath>$([System.IO.Path]::GetFullPath('$(MSBuildThisFileDirectory)../artifacts'))</ArtifactsPath>
+
+    <!--
+      Odd things can happen when WSL and Windows share the same output directories.
```

Comment:
maybe add `, i.e. when building on the same folder using both WSL and Windows`. Or just leave it and I can dig it from this comment.